packages/reports/tests/integration/components/filter-values/value-input-test.ts

```diff
@@ -4,12 +4,24 @@ import $ from 'jquery';
 import { render, fillIn } from '@ember/test-helpers';
 import { A as arr } from '@ember/array';
 import hbs from 'htmlbars-inline-precompile';
+import { TestContext as Context } from 'ember-test-helpers';
+import FilterFragment from 'navi-core/models/bard-request-v2/fragments/filter';
+import FragmentFactory from 'navi-core/addon/services/fragment-factory';
+//@ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
 
+interface TestContext extends Context {
+  filter: FilterFragment;
+  onUpdateFilter(changeSet: Partial<FilterFragment>): void;
+  isCollapsed: boolean;
+}
 module('Integration | Component | filter values/value input', function(hooks) {
   setupRenderingTest(hooks);
+  setupMirage(hooks);
 
-  hooks.beforeEach(async function() {
-    this.filter = { values: arr([1000]) };
+  hooks.beforeEach(async function(this: TestContext) {
+    const fragmentFactory = this.owner.lookup('service:fragment-factory') as FragmentFactory;
+    this.filter = fragmentFactory.createFilter('metric', 'bardOne', 'adClicks', {}, 'bet', [1000]);
```

Comment:
'bet' operator doesn't really make sense
```suggestion
    this.filter = fragmentFactory.createFilter('metric', 'bardOne', 'adClicks', {}, 'gt', [1000]);
```